variables.tf

```
@@ -110,6 +110,18 @@ variable "function_service_account_email" {
   description = "The service account to run the function as."
 }
 
+variable "function_vpc_connector" {
```

Since this is all part of a function module, `function_` seems redundant.

```suggestion
variable "vpc_connector" {
```